app/services/tweet_stat_adjustment_service.rb

```diff
@@ -0,0 +1,46 @@
+class TweetStatAdjustmentService
+  ADVANCE_IN_HOURS = 8
+
+  def initialize(content)
+    @content = content
+  end
+
+  def call
```

Comment:
I suggest you take a look at https://github.com/thepracticaldev/dev.to/blob/master/app/services/notifications/remove_each.rb to see how we're organizing the services code